src/components/ButtonComponents/NumberButtons/Numbers.js

```diff
@@ -1,16 +1,26 @@
-import React from "react";
+import React, { useState } from "react";
 
-//import any components needed
+// Import any components needed
+import NumberButton from './NumberButton';
+import { ButtonsContainer } from '../SpecialButtons/Specials';
 
-//Import your array data to from the provided data file
+// Import your array data to from the provided data file
+import { numbers } from '../../../data';
 
-const Numbers = () => {
+const Numbers = props => {
   // STEP 2 - add the imported data to state
+  const [calcNumbers, setCalcNumbers] = useState(numbers);
+
   return (
-    <div>
+    <ButtonsContainer>
       {/* STEP 3 - Use .map() to iterate over your array data and return a button
        component matching the name on the provided file. Pass
        it any props needed by the child component*/}
-    </div>
+       {calcNumbers.map(number => 
+         <NumberButton key={number} number={number} record={props.record} />
```

Comment:
Remember `key` is just a prop in this case, because you are applying it on a COMPONENT itself, `key` is only a reserved keyword, on a JSX element. For example: on a <div>, or <section>, etc